app/src/common/components/MovieList/MovieList.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import './MovieList.scss';
+
+import MovieElement from './MovieElement/MovieElement';
+
+export default class MovieList extends React.Component {
+  static defaultProps = {
+    movies: [],
+  };
+
+  static propTypes = {
+    movies: PropTypes.arrayOf(PropTypes.object),
+  };
+
+  render() {
+    if (!this.props.movies.length) {
+      return <div styleName="no-results">No films found</div>;
+    }
+
+    return (
+      <div styleName="list">
+        {this.props.movies.map(movie => <MovieElement key={movie.id} movie={movie} />)}
```

Comment:
I would implement it in a slightly different way.
```jsx
{this.props.movies.map(movie => <MovieElement key={movie.id} {...movie} />)}
```
In this case you don't pass the object by its reference. You provide values.

Reply:
done